src/result_collection.jl

```diff
@@ -88,7 +88,7 @@ function collect_results!(filename, folder;
         allfiles = String[]
         for (root, dirs, files) in walkdir(folder)
             for file in files
-                push!(allfiles, joinpath(root,file))
+                push!(allfiles, relpath(joinpath(root,file), datadir()))
```

Comment:
it might be worth it to make the relative directory a keyword argument, so that you have `rpath = projectdir()`, and use `rpath` in `relpath`.

Reply:
But I am not sure... At least that's how we do it in `tag!` where `gitpath` is given as argument., This way, if somneone really wants the absolute path, they can provide `""` as `rpath`.